code/__DEFINES/~skyrat_defines/cargo_import_companies.dm

```diff
@@ -7,7 +7,7 @@
 #define CARGO_COMPANY_KAHRAMAN (1<<5)
 #define CARGO_COMPANY_NAKAMURA_TOOLING (1<<6)
 #define CARGO_COMPANY_BOLT_NANOTRASEN (1<<7)
-#define CARGO_COMPANY_MICROSTAR (1<<8)
+#define CARGO_COMPANY_MICRON (1<<8)
```

Comment:
This wasn't just "an annoying change", MicroStar was a company combination introduced in https://github.com/Skyrat-SS13/Skyrat-tg/pull/17326, a project between Micron and AllStar to sell their weapons in one easy to reach place. It wasn't _just_ a rename, it was a way to combine Energy Guns into one single company for the imports system.

This PR not only reverts that, but seems to have entirely removed all other energy weapons from purchase. Removing the other energy guns from the system is not in the scope of this PR.